### ISSUE TITLE
Changes to website with PhET app release

### ISSUE THREAD

**Issue by amyh-phet** [phetsims/website#485]
To promote the release of the ipad app, changes need to be made to the website in three places.

1. Homepage - add a "Download at the app store" button to the table under Teaching Resources.
![front_page_ipad_app](https://cloud.githubusercontent.com/assets/13650180/17910139/d75f93a4-6944-11e6-9d18-1793cfda4cae.png)

2. Footer - add "Download at the app store" button to the footer.  
![footer_app_push](https://cloud.githubusercontent.com/assets/13650180/17910147/df7dacba-6944-11e6-935b-f178756c0221.png)

3. iPad/Tablet page - add screenshot of app and text to top of page to promote app.  Should be similar to layout of New Sims page, but will use a picture rather than video.
![ipad_page_app_push](https://cloud.githubusercontent.com/assets/13650180/17910158/e5b84a90-6944-11e6-9b17-e0df9711e01f.png)

@mattpen  Let me know if you need assets to make this happen.

**Comment by mattpen** [comment]
@amyh-phet @phet-steele - Could you please send a screenshot of the app for # 3?  There is about 324x220px worth of space there, the HTML5 video is 324x184px for comparision.  I've already downloaded the official download badge from the [developer's website](https://developer.apple.com/app-store/marketing/guidelines/#downloadOnAppstore).  

Is the copy in the 3rd screenshot final?

**Comment by amyh-phet** [comment]
@mattpenn   I can provide a screen shot for the app for #3.  I will verify that copy is final at website meeting today.  Should have an answer to you before 2 pm.

**Comment by amyh-phet** [comment]
@mattpen   Looks like we will be making changes to the screenshot/copy for the ipad/tablet page.   Currently working on this and will update once design is complete.

**Comment by amyh-phet** [comment]
@mattpen   We will be using an image that matches the artwork from the email campaign about the app release.  No copy needed - we will just give you an image to use that would fill the space in the mock up that is currently filled with the screenshot/copy.  What image size do you need?

**Comment by mattpen** [comment]
The same section on the NewSims page is 580x219px.  The height is a soft limit, but the width will be difficult to increase.

**Comment by amyh-phet** [comment]
Thanks.  The attached file gives you an idea but is not the final version of the image.  We will update once we know what the email will look like.    The text will be part of the image and doesn't need to be translatable.
![ipad_tablet_page_artwork](https://cloud.githubusercontent.com/assets/13650180/18105389/71940c62-6ebc-11e6-84e6-aa2151841a8b.png)


Also, since the app only has English sims, we were thinking the changes should only appear on the English version of the site.

**Comment by mattpen** [comment]
Looks nice!  Let me know when you have the final image.

**Comment by oliver-phet** [comment]
Here's a slightly updated mockup. I'm also going to send this to DU to see if he can put his touch on it!

Feedback?
![app_promotion_screen](https://cloud.githubusercontent.com/assets/5090275/18140112/547c1920-6f71-11e6-9c41-697dc7f30066.gif)

**Comment by amanda-phet** [comment]
I really like this version!!

**Comment by mattpen** [comment]
Just a few thoughts, fwiw.

It would be reasonably easy to detect if the user is browsing from an ipad and show this banner on other category pages for ipad users only.  I think it could work well on the new sims page and/or the html5 sims page since users are more likely to hit these pages before the by-device page.

It might also be nice to have the download button here: https://phet.colorado.edu/en/offline-access#ipad-header, and here: https://phet.colorado.edu/en/help-center/running-sims/tablet#q33-header

**Comment by oliver-phet** [comment]
I don't think we need to detect technology - some teacher use their computers to find sims for their classroom ipads. I think we will want to only show this in the _en locale, however.

I do like the idea of including it on the offline and help center pages! (Maybe only text for the help center)

Latest:
![app_website](https://cloud.githubusercontent.com/assets/5090275/18146947/af4adc1a-6f8f-11e6-9842-ae673bd91ad7.gif)

**Comment by oliver-phet** [comment]
Notes from status meeting:

1. Add gif to rotating image on homepage
2. Add promo image (when user detected on iPad) to 'New Sims', 'HTML5'
3. Always on 'iPad/Tablet'
4. Offline
5. Help Center (text)

**Comment by oliver-phet** [comment]
Here's a quick mockup for the homepage:
![app_website_homepage](https://cloud.githubusercontent.com/assets/5090275/18178771/eb5628b0-703c-11e6-90ac-7a6e97d1097d.gif)

**Comment by mattpen** [comment]
@oliver-phet - Did you intentionally remove the navigation bar?  It will take some time to implement a solution for one rotator item that is allowed to exceed the image bounds.  Its currently implemented by compiling one long image of all rotator items and hiding everything that isn't positioned above the nav bar.

**Comment by oliver-phet** [comment]
See updated screenshot above.

**Comment by ariel-phet** [comment]
Removing my assignment currently since I think I am happy with everything here

**Comment by oliver-phet** [comment]
Here's a new design incorporating some of Dave's feedback. This would go on:
2. Add promo image (when user detected on iPad) to 'New Sims', 'HTML5'
3. Always on 'iPad/Tablet'

Any feedback on this design? I think we may want to rework the text so that it can fit into the 'sim column'.
![app_promotion_ipad](https://cloud.githubusercontent.com/assets/5090275/18404657/203f8ac2-76a9-11e6-9b33-2e50c793eaeb.gif)

**Comment by amanda-phet** [comment]
I think the bullets could be rephrased so that they each have a similar flow:
* Natively designed for iPad 
* Ideal for offline use 
* Includes favorites, filtering, and automatic updates 
* Proceeds support development of new HTML5 sims

If they are too long, could they just go on to the next line?

**Comment by kathy-phet** [comment]
Shorter versions ... 

Native iPad experience
Ideal for offline use
Includes favorites and filtering
Proceeds fund new HTML5 sims

(apps usually update -- so perhaps automatic updates can just be presumed?)

**Comment by mattpen** [comment]
@amyh-phet 

> Also, since the app only has English sims, we were thinking the changes should only appear on the English version of the site.

I just wanted to confirm that this was still the plan.

**Comment by mattpen** [comment]
@oliver-phet - Could you send pngs or svgs for the rotator image and the banner image for the ipad page (even if they aren't finalized so I can put up placeholders on ox-dev)?

The rotator should be exactly 326x214px.  The banner image should be close to 520px in width, there isn't a hard limit on height.

**Comment by amyh-phet** [comment]
@mattpenn. As far as I know, that is the plan.

**Comment by oliver-phet** [comment]
@mattpen I'll send you the artwork shortly!

**Comment by oliver-phet** [comment]
Here are the 2 draft images.
![app_homepage](https://cloud.githubusercontent.com/assets/5090275/18603057/89fe2afe-7c2b-11e6-8630-592fec13d4b1.png)
![app_website](https://cloud.githubusercontent.com/assets/5090275/18603056/89fdf55c-7c2b-11e6-8eb0-24aa967c4a61.png)

**Comment by mattpen** [comment]
@oliver-phet - What should be displayed on the Offline Access page?  Just the download badge or one of the larger images?

**Comment by oliver-phet** [comment]
I would use the same iPad page image at the top of the Offline>iPad section.

**Comment by oliver-phet** [comment]
@mattpen It might make sense to try write parts of the iPad/Offline image in HTML. Do you think if we had images for the "the all new iPad app" and the iPad image, could you write the bullet points in HTML5. This would allow an actual button, rather than using the whole image as a button.

Thoughts?

**Comment by mattpen** [comment]
@oliver-phet - We could do the "the all new iPad app" text in html too.  This would allow us also to implement translatability once the app is internationalized.  

If the goal is simply making certain regions of the image clickable, we could use an image map instead.

**Comment by ariel-phet** [comment]
@oliver-phet @mattpen one opinion here. For the "the all new iPad app!" my preference would be "our all new iPad app!"

I feel like that tone just seem to fit us a bit better. But just my preference.

**Comment by oliver-phet** [comment]
@mattpen Here's a new png.
![app_website](https://cloud.githubusercontent.com/assets/5090275/18766429/7445f3d0-80d7-11e6-9848-8f902e513e93.png)

**Comment by oliver-phet** [comment]
Also a few other questions @mattpen:

1. The images on the site look a little pixelated... would it make sense to send you double resolution images? (I sent images of the exact resolution you specified).
2. Can we extend the duration for the iPad app rotator image? Maybe add 2 more seconds? (Not change the duration of the other Discipline gifs) If so, can we both remove the 'Download on the App Store' badge from the table on the homepage (under Teaching Resources)

**Comment by mattpen** [comment]
@oliver-phet 
1. Yes, it is ok to exceed the exact dimensions.
2. Yes, I can implement this.

**Comment by oliver-phet** [comment]
Some proposed text:

_Offline-access>iPad:_ 
We offer an iPad app that allows native offline use of all the HTML5 sims. Find it on the app store here.

In order to 'Download' an individual sim to use on an iPad (offline), you'll need to add it to the iPad reading list.

_Help Center>Running Sims>Tablet:_ 
**How do I run an HTML5 sim offline on an iPad?**
We offer an iPad app that allows native offline use of all the HTML5 sims. Find it on the app store here.

In order to 'Download' an individual sim to use on an iPad (offline), you'll need to add it to the iPad reading list.

_Help Center>Offline:_
**How do I run an HTML5 sim offline on an iPad?**
We offer an iPad app that allows native offline use of all the HTML5 sims. Find it on the app store here.

In order to 'Download' an individual sim to use on an iPad (offline), you'll need to add it to the iPad reading list.

**Comment by oliver-phet** [comment]
@mattpen here's a double resolution version of the images
![app_rotator](https://cloud.githubusercontent.com/assets/5090275/18795994/7e9b6d94-8184-11e6-81f7-0a1edd934268.png)
![app_website](https://cloud.githubusercontent.com/assets/5090275/18796049/ba86d0fa-8184-11e6-8dc5-98fae53a701a.png)

**Comment by amanda-phet** [comment]
For the help center questions can  the text 
>you'll need to add it to the iPad reading list. 

be phrased in a more positive way: you *can* add it to your iPad reading list.
?

Also, do we normally post the same question in two different places?

**Comment by oliver-phet** [comment]
I think it would be odd to phrase as "In order to... you can add it".

Maybe "You _can_ 'Download' an individual sim to use on an iPad (offline) _by adding_ it to the iPad reading list."

This is the only question (I believe) that is in two different places, but I thought it seemed so relevant to both?

**Comment by amyh-phet** [comment]
@mattpen    After looking at the changes to the homepage to promote the iPad app, we would like to remove the download badge from the Teaching Resources part of the table as the badge is also in the footer and rotating screenshots/video.

**Comment by oliver-phet** [comment]
_Help Center>Getting Started:_ 
**Do you have an app?**
We offer an iPad app that allows native offline use of all the HTML5 sims. Find it on the App Store here. Our app is $0.99 and proceeds fund more HTML5 sim development.

We currently do not offer an Android app.

**Comment by mattpen** [comment]
@oliver-phet - Should the help-center strings be shown in the english locale only? 

On the offline-access page, should the current text be shown in the english locale?  In the non-english locales?

**Comment by mattpen** [comment]
@amyh-phet @oliver-phet @amanda-phet - The latest changes are published to https://ox-dev.colorado.edu.

The help center questions are not up yet, those will need to be added through the admin tool once we put this in production.

**Comment by oliver-phet** [comment]
@mattpen Would it be easy to limit the help-center strings to English only? If that's the case, I think we only show them in the English locale. 

If it's a lot of work to do that, I think we might just modify the help-center strings to state that the app is 'English only'.

Eventually, when the app includes all translations, we'll want to include the app info on other website locales, however.

**Comment by mattpen** [comment]
@oliver-phet - It would be difficult to restrict help-center strings to one locale, I don't think it could be ready in time for the app store release.

**Comment by mattpen** [comment]
Links were added, deployment is on hold due to delay in publication of the app.

**Comment by mattpen** [comment]
@oliver-phet - The changes should all be live on phet-server now.  Can you review and let me know if we need anything else?  Also, can you please add the help center strings if they are finalized?

**Comment by oliver-phet** [comment]
Help center strings have been added but not all have updated yet.

**Comment by oliver-phet** [comment]
All strings are visible. Closing for now!
